conjure-undertow-processor/src/main/java/com/palantir/conjure/java/undertow/processor/data/ParamTypesResolver.java

```diff
@@ -59,9 +59,11 @@ public final class ParamTypesResolver {
             .collect(ImmutableSet.toImmutableSet());
 
     private final ResolverContext context;
+    private final DefaultDecoderNames defaultDecoderNames;
 
     public ParamTypesResolver(ResolverContext context) {
         this.context = context;
+        this.defaultDecoderNames = new DefaultDecoderNames(context);
```

Comment:
Should we just move `DefaultDecoderNames` into `ParamTypesResolver`? We don't instantiate `DefaultDecoderNames` anywhere else so it seems unnecessary for it to be a separate class.

Reply:
Entirely up to you